app/konnect/getting-started/import.md

```diff
@@ -20,110 +20,66 @@ Afterward, you must manually move over:
-
-1. Run [`deck dump`](/deck/latest/reference/deck_dump/) to export configuration into a file:
+1. Run [`deck gateway dump`](/deck/latest/reference/deck_gateway_dump/) to export the configuration into a file:
 
     ```sh
-    deck dump
+    deck gateway dump -o kong.yaml
     ```
 
     This command outputs {{site.base_gateway}}'s object configuration into
-    `kong.yaml` by default. You can also set `--output-file /path/{FILENAME}.yaml`
+    `-` to write to stdout by default. You can also set `--output-file /path/{FILENAME}.yaml`
     to set a custom filename or location.
 
-1. Open the file. If you have any of the following in your configuration, remove it:
-
-    * Any `_workspace` entries: There are no workspaces in {{site.konnect_short_name}}. For a similar
-    concept, see [control planes](/konnect/gateway-manager/control-plane-groups/).
-
-    * Configuration for the Portal App Registration plugin: App registration is
-    [supported in {{site.konnect_short_name}}](/konnect/dev-portal/applications/application-overview/),
-    but not through a plugin, and decK does not manage it.
-
-    * Any other unsupported plugins:
-        * OAuth2 Authentication
-        * Apache OpenWhisk
-        * Vault Auth
-        * Key Authentication Encrypted
-
-1. Preview the import with the [`deck diff`](/deck/latest/reference/deck_diff/)
+1. Preview the import with the [`deck gateway diff`](/deck/latest/reference/deck_gateway_diff/)
 command, pointing to the control plane that you want to target:
 
     ```sh
-    deck diff --konnect-control-plane-name default
+    deck gateway diff kong.yaml --konnect-token-file /PATH/TO/FILE
     ```
 
     If you're not using the default `kong.yaml` file, specify the filename and
     path with `--state /path/{FILENAME}.yaml`.
 
-1. If you're satisfied with the preview, run [`deck sync`](/deck/latest/reference/deck_sync/):
+1. If you're satisfied with the preview, run [`deck gateway sync`](/deck/latest/reference/deck_gateway_sync/):
 
     ```sh
-    deck sync --konnect-control-plane-name default
+    deck gateway sync kong.yaml --konnect-token-file /PATH/TO/FILE
     ```
 
     If you don't specify the `--konnect-control-plane-name` flag, decK targets the
     `default` control plane. If you have more than one control plane in your
     organization, we recommend always setting this flag to avoid accidentally
     pushing configuration to the wrong control plane.
 
-1. Log in to your [{{site.konnect_saas}}](http://cloud.konghq.com/login) account.
-
-1. From the left navigation menu, open **Gateway Manager**, then open the control plane
-you just updated.
+1. Log in to your {{site.konnect_saas}} account and open the control plane you just migrated in [**Gateway Manager**](https://cloud.konghq.com/gateway-manager/).
 
 1. Look through the configuration details of any imported entities to make sure
 they were migrated successfully.
 
 ## Migrate data planes
```

Comment:
Unrelated to what you wrote but I think this section should be removed. It doesn't actually give the user any information and detracts from the simplicity you are trying to achieve.